packages/client/src/utils/errors.tsx

```diff
@@ -8,6 +8,23 @@ import { tryFindMessageDescriptor } from "@gc-digital-talent/i18n";
 export const extractErrorMessages = (combinedError: CombinedError) =>
   combinedError.graphQLErrors.flatMap((error) => error.message);
 
+// grab the validation error enums to map them to messages, guarding against possible missing fields
+export const extractValidationMessageKeys = (
+  combinedError: CombinedError,
+): string[] | null => {
+  const { extensions } = combinedError.graphQLErrors[0];
+  if (extensions && extensions.validation) {
```

Comment:
Just a stylistic note but if you use optional chaining you can avoid the first check.
`if (extensions?.validation) {`